test/commands/reregister.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * test/commands/reregister.js
+ */
+'use strict';
+
+const expect = require('chai').expect;
+const nock = require('nock');
+const fork = require('child_process').fork;
+
+describe('test/commands/reregister >', () => {
+  const collectorName = 'collector1';
+  const refocusUrl = 'http://www.example.com';
+  const accessToken = 'abcdefghijklmnopqrstuvwxyz';
+  const invalidToken = 'aaa';
+  const collectorToken = 'zyxwvutsrqponmlkjihgfedcba';
+
+  const missingCollectorNameError = 'error: You must specify a collector name\n';
+  const missingUrlError = 'error: You must specify the url of the refocus instance\n';
+  const missingTokenError = 'error: You must specify an access token\n';
+
+  before(() => {
+    nock(refocusUrl, {
+      reqheaders: { authorization: accessToken },
+    })
+    .post('/v1/collectors/collector1/reregister')
+    .reply(201, { collectorToken });
```

Comment:
I think this should be 200. Also, using the constants from  `/src/constants.js` here and in line 40 , will make it clear that these are http status codes.